02_bubble/sarahdavy1983/main_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestMainOut(t *testing.T) {
+
+	main()
+
+	expected := []int{1, 2, 3, 5}
+	actual := []int{1, 2, 3, 5}
+
+	if !reflect.DeepEqual(expected, actual) {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+var tests = []struct {
+	name  string
+	input []int
+	want  []int
+}{
+	{"valid", []int{3, 2, 1, 5}, []int{1, 2, 3, 5}},
+	{"same number", []int{5, 5, 5, 5}, []int{5, 5, 5, 5}},
+	{"blank", []int{}, []int{}},
+	{"sorted", []int{1, 2, 3, 4}, []int{1, 2, 3, 4}},
+}
```

Comment:
Looks great to me!  My only suggestion would be to add a test case for if the array contains negative numbers :)

Reply:
Thanks!!  This is now updated :)